DRAFT_homogeneous_impact_areas.Rmd

```diff
@@ -0,0 +1,426 @@
+---
+title: "-"
+output:
+  html_document
+editor_options: 
+  markdown: 
+    wrap: sentence
+---
+
+# Homogeneous Impact Areas
+
+<br />
+
+*Author and date:* 
+
+Anders Kolstad
+
+```{r, echo=FALSE}
+Sys.Date()
+```
+
+<br />
+
+<!-- Load all you dependencies here -->
+
+```{r setup, include=FALSE}
+library(knitr)
+knitr::opts_chunk$set(echo = TRUE)
+library(sf)
+library(tmap)
+library(dplyr)
```

Comment:
Perhaps add a call to library(tidyverse) and get rid of two lines (dplyr and ggplot2) here.

Reply:
Also maybe consider adding in the version numbers to packages?

Reply:
Done